doc/google/google-drive-api.md

```diff
@@ -5,29 +5,48 @@ It uses `google-api-client` package which is [deprecated](https://github.com/goo
 ### Step1: A GCP project with the API enabled
 
 1. [create GCP project](https://developers.google.com/workspace/guides/create-project#create_a_new_google_cloud_platform_gcp_project)
-2. [enable Google Drive API and Google Calendar API](https://developers.google.com/workspace/guides/create-project#enable-api)
-3. Button of "Enable APIs and Services" is hard to find 
-
+![](https://i.imgur.com/qiTpYxr.png) （Should accept the Service Rule when first time using This）
```

Comment:
Should this png file be stored locally in `doc/images/` or somewhere like that?  I don't like the idea of images stored on someone's imgur account or elsewhere.

Reply:
same with all the other imgur links below.  that way we can also name the files something useful.

Reply:
I think we could change this in next branch

Reply:
agreed and that's a good idea! we can do that along with the new architecture diagrams.